examples/custom_route.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from aiohttp import web
+import typing
+
+from aiogram import Bot, Dispatcher, executor, types
+from aiogram.dispatcher.webhook import WebhookRequestHandler
+from aiogram.utils.executor import Executor
+from loguru import logger
+
+
+API_TOKEN = 'BOT TOKEN'
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+
+# Initialize bot and dispatcher
+bot = Bot(token=API_TOKEN)
+dp = Dispatcher(bot)
+
+
+@dp.message_handler(commands=['start', 'help'])
+async def send_welcome(message: types.Message):
+    await message.reply("Hi!\nI'm Custom Route Example bot!\nPowered by aiogram.\nSend any request to /ping route")
+
+
+@dp.message_handler()
+async def echo(message: types.Message):
+    await message.reply(message.text, reply=False)
+
+
+
+async def ping(request: web.Request):
+    '''
+        ping pong beep
+    '''
+    return web.Response(text="I`m ok!")
+
+async def who_am_i(request: web.Request):
+    '''
+        Return bot info from this route
+    '''
+    me = await request.app['BOT_DISPATCHER'].bot.me
+    return web.json_response(me.as_json())
+
+
+
+class CustomExecutor(Executor):
```

Comment:
Why exactly are we inheriting from the executor class? Is it to wrap `_web_app` initialized in parent executor?

Reply:
Yes, its for work directly with _web_app instance

Reply:
I don't think this is a good idea. Why don't you work with `aiohttp.web.Application` directly? It's little unclear as for me and would not be very obvious for beginners.